src/firecracker/src/main.rs

```diff
@@ -149,124 +153,114 @@ fn main_exec() -> Result<(), MainError> {
 
     let http_max_payload_size_str = HTTP_MAX_PAYLOAD_SIZE.to_string();
 
-    let mut arg_parser = ArgParser::new()
-        .arg(
-            Argument::new("api-sock")
-                .takes_value(true)
-                .default_value(DEFAULT_API_SOCK_PATH)
-                .help("Path to unix domain socket used by the API."),
-        )
-        .arg(
-            Argument::new("id")
-                .takes_value(true)
-                .default_value(DEFAULT_INSTANCE_ID)
-                .help("MicroVM unique identifier."),
-        )
-        .arg(
-            Argument::new("seccomp-filter")
-                .takes_value(true)
-                .forbids(vec!["no-seccomp"])
-                .help(
-                    "Optional parameter which allows specifying the path to a custom seccomp \
-                     filter. For advanced users.",
+    let mut arg_parser =
+        ArgParser::new()
```

Comment:
Could the formatting changes be moved to the "cargo fmt" commit?

[update: Hide whitespace in github makes it easier to see, but still would be best to split in 2]

Reply:
I think this is same as https://github.com/firecracker-microvm/firecracker/pull/4047#discussion_r1342429600 no?

Reply:
Yeah, don't know why GitHub posted it twice.